### PR TITLE
Openshift compatibility

### PR DESCRIPTION
**Pull request by nikita-akuity**
* Fix empty `imagePullPolicy`
* Remove `runAsUser`
* put redis `securityContext` into values
* Add `resources` to redis
```
+ diff /var/folders/5p/0vf6kdgd6zq36rhdcm10d_4r0000gn/T/tmp.q90mbj0u/helm_v2.6.7.yaml /var/folders/5p/0vf6kdgd6zq36rhdcm10d_4r0000gn/T/tmp.q90mbj0u/upstream_v2.6.7.yaml
17921c17921
<         image: quay.io/akuity/redis:7.0.9-alpine
---
>         image: redis:7.0.8-alpine
17973c17973
<         image: quay.io/akuity/redis:7.0.9-alpine
---
>         image: redis:7.0.8-alpine
18024c18024
<         image: quay.io/akuity/redis:7.0.9-alpine
---
>         image: redis:7.0.8-alpine
18052c18052
<         image: quay.io/akuity/redis:7.0.9-alpine
---
>         image: redis:7.0.8-alpine
18067a18068
>         fsGroup: 1000
18068a18070
>         runAsUser: 1000
```

### CODE REVIEW

File: charts/argo-cd/values.yaml [hunk 1]
```diff
@@ -289,14 +289,21 @@ redis:
   enabled: true
 
   image:
-    repository: redis
+    repository: quay.io/akuity/redis
     # https://hub.docker.com/_/redis/
-    tag: 7.0.8-alpine
+    tag: 7.0.4-alpine
```
Comment (jessesuen) [inline charts/argo-cd/values.yaml:294]: I misspoke earlier. We should use:

```suggestion
    tag: 7.0.9-alpine
```

**Comment by evgeny-goldin** [comment]
Could you run the compare script (in ./hack) and add it's output to the change? Normally, we expect the output to be clean but this time we probably expect it to show the diff introduced from the upstream.

**Comment by nikita-akuity** [comment]
@evgeny-goldin updated the PR description

**Review by jessesuen — APPROVED** [review]
LGTM. Thank you!

**Review by evgeny-goldin — APPROVED** [review]

**Pull request merged**